assets/js/components/chess-board.js

```diff
@@ -41,29 +68,71 @@ class ChessBoard extends React.Component {
   renderRanks() {
     const board = this.getBoard();
 
-    return _.map(Object.keys(board).reverse(), (rankId) => {
+    return _.map(this.ranks(), rankId => {
       return (
-        <tr className="board-rank" key={rankId}>
+        <div className="board-rank" key={rankId}>
```

Comment:
Any reason this `div` is added here as opposed to in the `ChessBoardSquare` component?